test/compile.jl

```diff
@@ -32,8 +39,12 @@ try
 
     # use _require_from_serialized to ensure that the test fails if
     # the module doesn't load from the image:
-    println(STDERR, "\nNOTE: The following 'replacing module' warning indicates normal operation:")
```

Comment:
i think we were generally not really checking whether it broke previously either (unless someone was reading through the logs and noticed that the error wasn't the expected one)